x-pack/plugin/otel-data/src/main/resources/component-templates/metrics-otel@mappings.yaml

```diff
@@ -15,3 +15,33 @@ template:
         type: keyword
         time_series_dimension: true
         ignore_above: 1024
+    dynamic_templates:
+      - histogram:
+          mapping:
+            type: histogram
+            ignore_malformed: true
+      - counter_long:
+          mapping:
+            type: long
+            time_series_metric: counter
+            ignore_malformed: true
+      - gauge_long:
+          mapping:
+            type: long
+            time_series_metric: gauge
+            ignore_malformed: true
+      - counter_double:
+          mapping:
+            type: double
+            time_series_metric: counter
+            ignore_malformed: true
+      - gauge_double:
+          mapping:
+            type: double
+            time_series_metric: gauge
+            ignore_malformed: true
+      - summary_metrics:
```

Comment:
Should we take the opportunity to make this one's name consistent with the others?

Reply:
Makes sense to me. We'd then just call it `summary`, right? This means that there will need to be changes in the ES exporter, though.

cc @carsonip

Reply:
Yes, I was thinking `summary` too.

Reply:
I renamed it to `summary`. @carsonip I'll wait for you to acknowledge this before I push merge, just to make sure we don't miss the change in ES exporter.

Reply:
ack. Will rename it to `summary`.

> I'm wondering if rolling out that change is easier by temporarily adding both summary and summary_metrics in this PR, then making the change in the ES exporter, then removing the summary_metrics dynamic template in another ES PR.

I'm not too concerned. e2e poc pins ES version and es exporter version anyway. The mains don't have to be compatible at every point of time.

Reply:
PR: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35424